### DISCUSSION TITLE
GTK for Web Developers

### DISCUSSION THREAD

**Discussion by dar5hak** [elementary/edw#16] [Accepted Talks]
Speaker: @dar5hak

Bridge your web development know-how to the desktop world. This draws parallels between CSS Grid and `Gtk.Grid`, DOM tree and `Gtk.Window` hierarchy, and (loosely) reactive components and `Gtk.Widget`. Also mentions what does not carry over from the web (like declarative markup).

This could also touch upon GTK CSS, unless another talk (like #7) plans to cover that.

**Comment by clagosarias** [comment]
This sounds really really interesting :ok_hand:

**Comment by techdev5521** [comment]
I'd very much be interested in this as this is the very path I am on now!

**Comment by mr-islam** [comment]
Would really love this myself, and it should help a large audience who want to make the jump from web dev into GTK

**Comment by niutech** [comment]
There is also [GJS guide](https://gjs.guide/) for those interested.

**Comment by megatux** [comment]
It would be nice if it explains the Elementary design choices inside the GTK CSS definitions, both light & dark themes

**Reply by dar5hak** [reply]
That's something I am yet to learn too. Let me see what fits into the scope for the target audience.

**Comment by cassidyjames** [comment]
@dar5hak we've accepted your talk. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.

Another interesting thing to cover (depending on time) is using the GTK inspector and how it's similar to web inspector tools.

**Reply by dar5hak** [reply]
Thanks, Cassidy. I'll start outlining the content.

Yeah, GTK inspector would be a suitable addition. I'll introduce that as well. 👍
